_docs_custom_metrics/generic-script/index.md

```diff
@@ -4,128 +4,174 @@ title: Generic script plugin
 description: Information on how to use the CoScale generic script plugin.
 ---
 ## Description
-Once you have an [agent]({{ site.baseurl }}/agent/index) running on your server, you can add the generic script plugin to gather custom metrics and events. The plugin executes a script once every minute to gather data from it and sends it to CoScale.
+Once you have an [agent]({{ site.baseurl }}/agent/index) running on your server, you can add the generic script plugin to gather custom metrics and events. The plugin executes the scripts you provide once every minute to gather data from it and sends it to CoScale.
 
-The generic script plugin has **2** modes: **configuration** mode and **data retrieval** mode. Our plugin will first run in configuration mode to get the metrics that should be created. Afterwards, it will run every minute in data retrieval mode to get the data.
-
-{% include alert.html type="warning" text="Scripts have a maximum runtime of 5 seconds." %}
+The scripts provided to the generic script plugin must have **2** modes: a **configuration** mode and a **data retrieval** mode. Our plugin will first run your scripts in configuration mode to get the metrics that should be created. Afterwards, it will run the scripts every minute in data retrieval mode to get the data.
```

Comment:
'it will run the scripts every minute'

Is only the case when you use period 60. "Afterwards, it will run the scripts every period of time (defined in configuration mode) in data retrieval mode to get the data."